application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -642,24 +674,35 @@ define('diagram-link-editor', [
           str = $('&lt;div/&gt;').html(str).text() || this.foAltText;
         }
 
-        var previousHeight;
         if (this.foAltText != null) {
           var s = this.state;
           var htmlConverter = document.createElement('textarea');
           var content = document.createElementNS('http://www.w3.org/2000/svg', 'g');
-          var elements = fo.firstElementChild.firstElementChild.childElements();
+          var elementsContainer = $(fo).children().first().children().first();
+          var elements = elementsContainer.contents();
```

Comment:
From the documentation of ``contents()``:

> Get the children of each element in the set of matched elements, **including text and comment nodes**.

So if your variable is called ``elements`` then I assume you want only the elements and thus you should use ``children()`` instead. If you need to process all nodes, including text and comment nodes, then you should rename the variable.

Reply:
I do need to process all nodes, so I'll do the rename